src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
you can set loading to true initially, and then unset it when your data is fetched